spec/features/admin/division_flow_spec.rb

```diff
@@ -12,4 +13,12 @@
   include_examples :flow do
     subject { division }
   end
+
+  scenario "division name and parent division can't be the same" do
+    visit admin_division_path(division)
+    find('.edit-action').click
+    select 'Cream', from: 'division_parent_id'
+    click_on 'Update Division'
+    expect(page).to have_content('Parent Division and Name can not have the same value')
```

Comment:
Update to "Division and Parent Division cannot be the same" or use the yml content.